bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "raygun4js",
-  "version": "2.26.6",
+  "version": "2.26.7",
```

Comment:
Might be worth a minor version bump, rather than a patch? 

It's a fairly large, but not breaking, change in how timings are fetched